homeassistant/components/frontier_silicon/__init__.py

```diff
@@ -1 +1,38 @@
-"""The frontier_silicon component."""
+"""The Frontier Silicon NEW integration."""
```

Comment:
Why call it the 'new integration'?

Reply:
Forgot to change this back to the original (was to distinguish between the old and new implementation).